command/agent/agent_endpoint.go

```diff
@@ -36,6 +36,11 @@ func (s *HTTPServer) AgentSelf(resp http.ResponseWriter, req *http.Request) (int
 	}, nil
 }
 
+func (s *HTTPServer) AgentReload(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
```

Comment:
Let's make these new ones require a PUT otherwise give a 405. Might as well start these off in a better spot :-)